opus/application/static_media/js/hash.js

```diff
@@ -18,20 +18,21 @@ var o_hash = {
 
     // updates the hash according to user selections
     updateHash: function(updateURL=true) {
-
         let hash = [];
         for (let param in opus.selections) {
             if (opus.selections[param].length) {
-                hash.push(param + "=" + opus.selections[param].join(",").replace(/ /g,"+"));
+                let encodedSelectionValues = o_hash.encodeSlugValue(opus.selections[param]);
+                hash.push(param + "=" + encodedSelectionValues.join(","));
             }
         }
 
         for (let key in opus.extras) {
+            let encodedExtraValues = o_hash.encodeSlugValue(opus.extras[key]);
             try {
```

Comment:
Since encodeSlugValue always returns an array now, we probably don't need this try anymore.